src/Sulu/Bundle/MediaBundle/Media/Storage/LocalStorageInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\Storage;
+
+interface LocalStorageInterface extends StorageInterface
+{
+    /**
+     * Returns the path for the given file.
+     */
+    public function getLocalPath(array $storageOption): ?string;
```

Comment:
Maybe I don't understand all the neccessities here, but in general I would prefer a consistent interface for all storages.